config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+common: &common
+  secret_key_base: 'f0b774b4d06f7ea3357dec260cd6231367009d35525b3c437b27e299d52599bebd4eb1cd83fc3f69ff0d46a0510e5e4ba3bffd26b503b7713100f6083fc4baba'
+
+  smtp:
+    user_name: <%= ENV['SMTP_USER_NAME'] || 'owen' %>
```

Comment:
Previously it was hardcoded in [config/environments/development.rb](https://github.com/ywwg/ffagc/blob/79f34ce4a13e9040ce5c180bca62a6e88d1d4ff3/config/environments/development.rb#L54-L60)